styles.css

```diff
@@ -1,785 +1,862 @@
-      font-style: normal;
-      font-weight: 500;
-      font-size: 4.53vw;
-      line-height: 8vw;
-      display: flex;
-      align-items: center;
-      text-align: center;
-      letter-spacing: 0.03em;
-      color: #396DF2;
-      flex: none;
-      order: 0;
-      flex-grow: 0;
-      border: #396DF2 solid 1.5px;
-      background-color: white;
-      border-radius: 8px;
-      padding: 1vw 2vw 1vw 2vw;
-      cursor: pointer;
-      margin-top: 2.5vw;
-      margin-left: 7vw;
-    }
-    .touch:hover{background-color: #396DF2; border:  #FFFFFF solid 2px;color:  #FFFFFF;}
-    .touch:active{background-color: #FFFFFF;color: #000000;border:  #FFFFFF solid 2px;}
-    .foot{
-      width: 27.5vw;
-      height: 1vw;
-      margin-left: 30vw;
-      margin-bottom: 2vw;
-      background: #0D151E;
-      border-radius: 10px;
-      margin-top: 7.5vw;
-    }
\ No newline at end of file
+@media only screen and (min-width: 768px) {
+  .body{
+    background: #FFFFFF;
+  }
+    .navigation{
+        width: 100vw;
+        display: flex;
+        flex-direction: row;
+        flex-wrap: nowrap;
+        position: fixed;
+    }
+    .mylogo{
+        width: 9.89vw;
+        height: 5vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 700;
+        font-size: 2.34vw;
+        line-height: 5vw;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #6070FF;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        padding-left: 10vw;
+    }
+    .portfolio{
+        width: 8.07vw;
+        height: 5vw;
+        font-family: 'Poppins', sans-serif;;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 1.95vw;
+        line-height: 5vw;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #344563;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 40vw;
+    }
+    .about{
+        width: 8.07vw;
+        height: 5vw;
+        font-family: 'Poppins', sans-serif;;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 1.95vw;
+        line-height: 5vw;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #344563;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 5vw;
+    }
+    .contact{
+        width: 8.07vw;
+        height: 5vw;
+        font-family: 'Poppins', sans-serif;
+        font-style: normal;
+        font-weight: 500;
+        font-size: 1.95vw;
+        line-height: 5vw;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #344563;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+        margin-left: 5vw;
+    }
+    .page{
+        display: grid;
+        grid-template: 100% 100%;
+        background: #E5E5E5;
+        gap: 10vw;
+    }

```

Comment:
- [ ] Please make sure you stick to the Figma design when choosing the background color of the page as the background is specified as `#f4f5f7` in the Figma design and in your design it's `#E5E5E5` so please make sure to update this. 
- Your design:
  ![image](https://github.com/feyisakedir/Portfolio-2nd-Project/assets/104968236/4b9b14fa-e3c9-4400-836e-0cfc37ab60fd)